docs/getting-started-interfaces.md

```diff
@@ -0,0 +1,45 @@
+# How do I get started, what interfaces can I use?
+
+To interact with a model it is recommend that you utilize one of the pre-existing interfaces that have been built by the open-source community.
+
+These interfaces provide several handy features:
+- A chat-like interface to interact with your model
+- Swapping between prompt types to match the model being used
+- Model re-training (currently Ooba only)
+- Support for changing model parameters on the fly
+
+There are a few interface options based on your particular needs:
+
+##### text-generation-webui (aka Oobabooga)
+
```

Comment:
Generally, I think the UI's descriptions should be separated by what the difference is for the user, not the architecture. IE oobabooga supports the widest variety of models and gives you lots of control/ Koboldcpp has a clean user interface that makes it easy to get started.

Something along the lines of that. The features that would make a difference to someone who's never used this stuff before.

Reply:
Updated the layout based on this feedback.